src/mappings/curveSETHVaultMappings.ts

```diff
@@ -106,3 +113,90 @@ export function handleUpdateManagementFee(
     );
   }
 }
+
+export function handleUpdateGuardian(event: UpdateGuardian): void {
```

Comment:
If you wanted to cut down on the amount of code repetition for handleUpdateGuardian/handleUpdateManagement/handleUpdateGovernance/handleUpdateDepositLimit, I recommend abstracting the code away to src/utils/vault/vault.ts. 

It's best to keep the code in the mapping files as brief as possible since it has to be repeated in so many places. Here's a very similar example for performanceFeeUpdated; all we have to do in the mapping.ts files is prepare the args and call the function here: https://github.com/yearn/yearn-vaults-v2-subgraph/blob/main/src/utils/vault/vault.ts#L465-L495